modules/network-observability-lokistack-create.adoc

```diff
@@ -7,6 +7,9 @@
 = Creating a LokiStack custom resource
 You can deploy a LokiStack using the web console or CLI to create a namespace, or new project.
 
+include::snippets/logging-clusteradmin-access-logs-snip.adoc[]
+For more information about creating a `cluster-admin` group, see the "Additional resources" section.  
```

Comment:
Is it on purpose to redirect readers to another section from where they're again redirected via a link? Wouldn't it be simpler to have that link here in the first place?

Reply:
Agreed

Reply:
It is on purpose :( Its a sad docs experience reality where we can only create cross references to `MODULES` in an `ASSEMBLY` files. We're prevented from creating cross references in `MODULES`. It breaks the builds. All we can do is create "Additional resources" sections that follow the file inclusion in the `ASSEMBLY` and link to the file. The reason for this is part of my question, and Nathan's in the Jira, our modular docs people say that if we are going to link we might as well just add the module outright. There's a lot of disagreement about this because sometimes you might just want to link  and not clutter the page with a bunch of content that may or may not apply to a user. and that's OK.

Reply:
I think that once this LokiStack resources table is out of the way ([PR](https://github.com/openshift/openshift-docs/pull/67435) to make this happen), the Additional Resources will be more properly placed. Right now, that table is in the way.